ansible/roles_ocp_workloads/ocp4_workload_vertical_pod_autoscaler/defaults/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+silent: false
+
+# Channel to use for the OpenShift vertical_pod_autoscaler subscription
+ocp4_workload_vertical_pod_autoscaler_channel: '4.8'
+
+# Set automatic InstallPlan approval. If set to false it is also suggested
+# to set the starting_csv to pin a specific version
+# This variable has no effect when using a catalog snapshot (always true)
+ocp4_workload_vertical_pod_autoscaler_automatic_install_plan_approval: true
+
+# Set a starting ClusterServiceVersion.
+# Recommended to leave empty to get latest in the channel when not using
+# a catalog snapshot.
+# Highly recommended to be set when using a catalog snapshot but can be
+# empty to get the latest available in the channel at the time when
+# the catalog snapshot got created.
+ocp4_workload_vertical_pod_autoscaler_starting_csv: ""
+
+# --------------------------------
+# Operator Catalog Snapshot Settings
+# --------------------------------
+# See https://github.com/redhat-cop/agnosticd/blob/development/docs/Operator_Catalog_Snapshots.adoc
+# for instructions on how to set up catalog snapshot images
+
+# Use a catalog snapshot
+ocp4_workload_vertical_pod_autoscaler_use_catalog_snapshot: false
+
+# Catalog Source Name when using a catalog snapshot. This should be unique
+# in the cluster to avoid clashes
+ocp4_workload_vertical_pod_autoscaler_catalogsource_name: redhat-operators-snapshot-vertical_pod_autoscaler
+
+# Catalog snapshot image
+ocp4_workload_vertical_pod_autoscaler_catalog_snapshot_image: quay.io/gpte-devops-automation/olm_snapshot_redhat_catalog
+
+# Catalog snapshot image tag
+ocp4_workload_vertical_pod_autoscaler_catalog_snapshot_image_tag: v4.8_2021_7_15
```

Comment:
07, not 7

Reply:
thanks.  oy vey.  trying to get out of the house to bring the little ones to the doc.. :-)